plenum/test/node_catchup_with_3pc/test_pp_obsolescence_check_fail_for_delayed.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import asyncio
+
+from plenum.test import waits
+from plenum.test.delayers import ppDelay, pDelay, cDelay
+from plenum.test.helper import sdk_send_random_and_check
+from plenum.test.node_request.test_timestamp.helper import get_timestamp_suspicion_count
+from plenum.test.node_catchup.helper import ensure_all_nodes_have_same_data
+from plenum.test.stasher import delay_rules
+from plenum.test.spy_helpers import get_count
+from stp_core.loop.eventually import eventually
+
+
+# TODO better to use non hard coded value of nodes number
```

Comment:
`nodeCount` can be used in any test to set the number of nodes.

Reply:
Thanks